engine/eConstr.ml

```diff
@@ -727,7 +735,30 @@ let closed0 sigma c = closedn sigma 0 c
 
 let subst_of_rel_context_instance ctx subst =
   cast_list (sym unsafe_eq)
-    (Vars.subst_of_rel_context_instance (cast_rel_context unsafe_eq ctx) (cast_list unsafe_eq subst))
+    (Vars.subst_of_rel_context_instance (cast_rel_context unsafe_eq ctx) (cast_vect unsafe_eq subst))
+let subst_of_rel_context_instance_list ctx subst =
+  cast_list (sym unsafe_eq)
+    (Vars.subst_of_rel_context_instance_list (cast_rel_context unsafe_eq ctx) (cast_list unsafe_eq subst))
+
+let liftn_rel_context n k ctx =
```

Comment:
There are stray spaces after `unsafe_eq` in all the functions below.

Reply:
Ah yes, will remove.